api/pom.xml

```diff
@@ -59,6 +59,10 @@
 
     <build>
         <plugins>
+            <plugin>
```

Comment:
Is this plugin config needed here?

Reply:
Yes.
We don't need it in both modules: `api` and `impl`. Because it is enough for us to have it executed only once.
And we should it have here (but not in `impl`) as `api` module will be built before `impl` https://github.com/okta/okta-sdk-java/blob/dc7ac324de111c8f71a22c3e53023d5c9cf973d4/pom.xml#L52